### ISSUE TITLE
Home: Links do not have sufficient contrast /missing underline

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#4]
**URL**: https://hcdev.openplus.ca/en
**User Agent**: Chrome/Firefox
**Language**: both

**Issue Description:**  Links in the main content of the page are almost black and don't have enough ratio with surrounding text. Some of the links do have underline but no visual colour for links. Some links are missing underline as well. There is also no change with mouse hovering.

Please see screenshot of some of the links. 
![image](https://user-images.githubusercontent.com/65728700/126988183-a94b4392-19fa-4692-98c9-57a8a1258eb1.png)

**Steps to reproduce**: Visual, keyboard tabbing, mouse hovering

**Expected Results**: The links should be visually evident. There should be a contrast ratio of 3:1 of links with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them.

**Comment by davidgotlieb** [comment]
﻿fixed, style matches Cananda design style ﻿https://www.canada.ca/en/government/about/design-system.html

**Comment by mmiron-HC** [comment]
Re: contrast - links are on their own and not within a body of text AND are now being displayed with an underline, therefore there is no requirement to make them higher contrast. 

Closing issue - Hovering will be address via a new issue.
